sites/all/themes/scholarly_lite/style.css

```diff
@@ -253,7 +259,7 @@ h1.title { margin-top:0; margin-bottom: 20px; }
 
 #highlighted + #main-content { padding: 20px 0; }
 
-#main-content { padding: 60px 0; }
+#main-content { padding: 0px 0; }
```

Comment:
This doesn't need to be changed, but for future reference in CSS you don't have to explicitly state `0px` you can just use `0` instead. It isn't much of a difference, but it is good to keep a consistent style with the rest of the file where possible (e.g. `padding: 0 0;` or even just `padding: 0;`).